### ISSUE TITLE
Add installation instructions

### ISSUE THREAD

**Issue by GiuseppeGalilei** [ekallevig/garmin-spotify-widget#2]
This seems like a great project! 
I really want to try it out, but I don't know where to begin 😅
Can you provide some hints about the installation and setup process?

**Comment by RTeran** [comment]
Yes, I want to try it too!
Could you give us some installation instructions?

**Comment by ekallevig** [comment]
Unfortunately garmin introduced a bug in their sdk that forces a particular value for the redirect_uri param in the oauth requests, which causes auth to fail as spotify requires them to match on both /authorize and /api/token request. https://forums.garmin.com/developer/connect-iq/i/bug-reports/makewebrequest-after-makeoauthrequest-is-caching-redirect_uri-param

**Comment by RTeran** [comment]
I see... what a pity...
So, is the problem with the SDK or with the Garmin Connect app?
Have you tried the trick of using the string it returns ("gcm://oauth") as the redirect_uri to see if it works with that value? Maybe, if all Spotify needs is for the two redirect_uris to be the same, you can set the redirect_uri to "gcm://oauth" in the Spotify App settings.
I don't know... just to try...

**Comment by RTeran** [comment]
I just tried it and it doesn't work. Of course, it doesn't make sense.

**Comment by ekallevig** [comment]
> I see... what a pity...
> 
> So, is the problem with the SDK or with the Garmin Connect app?
> 
> Have you tried the trick of using the string it returns ("gcm://oauth") as the redirect_uri to see if it works with that value? Maybe, if all Spotify needs is for the two redirect_uris to be the same, you can set the redirect_uri to "gcm://oauth" in the Spotify App settings.
> 
> I don't know... just to try...

Yeah I did try this a while ago and just tried  again with the latest SDK but it doesn't know where to redirect to after auth. Strange bug as the actual oauth request is fine but then regular web requests are overriding this param.
